packages/skeleton/src/lib/components/Paginator/Paginator.svelte

```diff
@@ -150,7 +163,7 @@
 		{#if showFirstLastButtons}
 			<button
 				type="button"
-				class={buttonClasses}
+				class="{buttonClasses} 0"
```

Comment:
Was this a typo?

Reply:
ops, yeah sorry this was a typo, I will fix it.